crates/cli/src/bin/wasm-bindgen-test-runner/headless.rs

```diff
@@ -251,11 +251,13 @@ impl Driver {
     /// extra arguments to the driver's invocation.
     fn find() -> Result<Driver, Error> {
         let env_args = |name: &str| {
-            env::var(format!("{}_ARGS", name.to_uppercase()))
-                .unwrap_or_default()
-                .split_whitespace()
-                .map(|s| s.to_string())
-                .collect::<Vec<_>>()
+            let var = env::var(format!("{}_ARGS", name.to_uppercase())).unwrap_or_default();
+
+            shlex::split(&var).unwrap_or_else(|| {
+                var.split_whitespace()
+                    .map(|s| s.to_string())
+                    .collect::<Vec<_>>()
```

Comment:
```suggestion
                    .collect()
```
This shouldn't be needed anymore.